client/signup.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+//import Login from './login.jsx';
+import ReactDOM from 'react-dom';
+import Signin from './signin.jsx';
+//import'whatwg-fetch';
+
+function postSend(url, data) {
+  return fetch(url, {
+    credentials: 'same-origin',
```

Comment:
do same thing as I said in the other file. fix padding, remove comments and empty lines